stdlib/REPL/src/REPL.jl

```diff
@@ -1404,9 +1404,9 @@ end
 
 function out_transform(@nospecialize(x), n::Ref{Int})
     return quote
-        let x = $x
-            $capture_result($n, x)
-            x
+        let __temp_val_a72df459 = $x
+            $capture_result($n, __temp_val_a72df459)
+            __temp_val_a72df459
         end
```

Comment:
Alternatively, I had toyed with making `capture_result` return the value too, so that it could be written as:
```suggestion
        $capture_result($n, $x)
```